src/app/loadout-drawer/LoadoutDrawerContents.tsx

```diff
@@ -17,9 +18,10 @@ import { addIcon, AppIcon } from '../shell/icons';
 import { Loadout } from './loadout-types';
 import LoadoutDrawerBucket from './LoadoutDrawerBucket';
 import SavedMods from './SavedMods';
+import SavedSubclass from './SavedSubclass';
+import SubclassDrawer from './subclass-drawer/SubclassDrawer';
 
 const loadoutTypes: DimBucketType[] = [
-  'Class',
```

Comment:
Possibly just my understanding of the code, I will have another look over it. I thought this determined what buttons we have for the item picker to show. I now have a custom button action so I add the Class button manually.

Reply:
Yeah so `loadoutTypes` determines what buttons are rendered and what selected `DimItems` are rendered using the old layout.

I have purposely removed it because
- I render the subclass with it's mods, so the old display of only showing the class item is gone.
- I make sure the button to open the subclass drawer is always available.

We could instead have special handling for class cases where these types are iterated over (they are split into types with loadout items and ones without). I don't have a strong opinion either way so happy to go with your preference.

Reply:
Makes sense! One thing to keep in mind is that it's possible to choose three subclasses in a loadout (one per class)

Reply:
Ah really? I can't figure out how to do that. If so I haven't handled that.

Reply:
Figured it out after a chat, I will add this in.